cfy_manager/components/restservice/scripts/create_tables_and_add_defaults.py

```diff
@@ -179,10 +208,19 @@ def _add_provider_context(context):
 
     if script_config.get('db_migrate_dir'):
         _init_db_tables(script_config['db_migrate_dir'])
-    if script_config.get('admin_username') and \
-            script_config.get('admin_password'):
+    if (script_config.get('admin_username')
+            and script_config.get('admin_password')):
```

Comment:
btw if this throws a flake warning for you, you need to update your flake (hi @geokala )

Reply:
No error :man_shrugging:

Reply:
Just wanted to remove the `\` @tyacbovi